src/Application/Actions/Donations/Get.php

```diff
@@ -39,7 +40,7 @@ protected function action(Request $request, Response $response, array $args): Re
         if (!$donation) {
             throw new DomainRecordNotFoundException('Donation not found');
         }
-
+        $donation->setDonationStatus(DonationStatus::Paid);
```

Comment:
```suggestion
```

These lines must have been committed by mistake. I tend to do the same on my local when I'm testing frontend stuff to make the thank you page load. But probably we should have a unit test that will fail when these lines are here - I might add one.